psi4/src/psi4/fnocc/df_ccsd.cc

```diff
@@ -418,6 +422,9 @@ PsiReturnType DFCoupledCluster::CCSDIterations() {
   outfile->Printf("        D1 diagnostic:                  %20.12lf\n",sqrt(eigval->pointer()[0]));
   outfile->Printf("\n");
 
+  // add D1 diagnostic to globals
+  Process::environment.globals["CC D1 DIAGNOSTIC"] = sqrt(eigval->pointer()[0]);
+
```

Comment:
OK, I'll remove those lines for now.  I should have time to duplicate them in the whole module later this week.